plugin/src/main/groovy/com/novoda/staticanalysis/EvaluateToolViolationsTask.groovy

```diff
@@ -0,0 +1,20 @@
+package com.novoda.staticanalysis
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+class EvaluateToolViolationsTask extends DefaultTask {
+
+    Closure<ViolationsEvaluator> evaluator
+    Violations toolViolations
+
+    EvaluateToolViolationsTask() {
+        group = 'verification'
+        description = "Evaluate violations against penaltyExtension thresholds."
```

Comment:
Since it is per-tool, It is a good practice to mention the tool name in the description.